openlibrary/plugins/upstream/account.py

```diff
@@ -325,7 +325,11 @@ def POST(self):
         if error:
             return self.render_error(error, i)
 
-        expires = (i.remember and 3600 * 24 * 7) or ""
+        expires = i.remember and 3600 * 24 * 7
+
+        if email in ['mek@archive.org', 'brenton@archive.org', 'brewster@archive.org']:
```

Comment:
This cookie set + email check has to be done within L321 audit_accounts (using xauthn response)

L330 is a stub to unblock the rest of the UI